### DISCUSSION TITLE
.NET issue

### DISCUSSION THREAD

**Discussion by XReachlingX** [num0005/Osoyoos-Launcher#26] [General]
![Screenshot 2022-02-06 172612](https://user-images.githubusercontent.com/93212762/152706118-73f1bdff-07c6-4259-b3d2-4cd440f3da28.png)

I have .NET 5.0 on my PC but every time I try to open it, this notification keeps popping up

**Comment by NineToFiveG** [comment]
Probably a bit late, but if you haven't found the fix for this yet, the solution is in the README.txt in the code tab. You have to manually download the correct version of microsoft's 5.0 .NET, as for some reason it just refuses to do it through that window prompt.
